src/engine/Actor.ts

```diff
@@ -5,8 +5,8 @@ import { Texture } from './Resources/Texture';
 import {
```

Comment:
There are a lot of empty whitespace removal changes, can you undo these in files/areas that aren't related to the change?

A separate PR focused on correcting trailing whitespace would be great.